tests/monitoring/component_monitoring.go

```diff
@@ -161,34 +161,48 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 	})
 
 	Context("Errors metrics", func() {
-		var crb *rbacv1.ClusterRoleBinding
+		var crbBackups []*rbacv1.ClusterRoleBinding
+		var crbs = []string{
+			"kubevirt-operator",
+			"kubevirt-controller",
+			"kubevirt-handler",
+		}
 
 		BeforeEach(func() {
 			virtClient = kubevirt.Client()
 
-			crb, err = virtClient.RbacV1().ClusterRoleBindings().Get(context.Background(), "kubevirt-operator", metav1.GetOptions{})
-			util.PanicOnError(err)
-
 			increaseRateLimit()
 
+			for _, crb := range crbs {
+				crb, err := virtClient.RbacV1().ClusterRoleBindings().Get(context.Background(), crb, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				crbBackups = append(crbBackups, crb.DeepCopy())
+			}
+
 			scales = libmonitoring.NewScaling(virtClient, []string{virtOperator.deploymentName})
 			scales.UpdateScale(virtOperator.deploymentName, int32(0))
 
 			libmonitoring.ReduceAlertPendingTime(virtClient)
 		})
 
 		AfterEach(func() {
-			crb.Annotations = nil
-			crb.ObjectMeta.ResourceVersion = ""
-			crb.ObjectMeta.UID = ""
-			_, err = virtClient.RbacV1().ClusterRoleBindings().Create(context.Background(), crb, metav1.CreateOptions{})
-			if !errors.IsAlreadyExists(err) {
-				util.PanicOnError(err)
+			for _, crb := range crbBackups {
+				crb.Annotations = nil
+				crb.ObjectMeta.ResourceVersion = ""
+				crb.ObjectMeta.UID = ""
+				_, err = virtClient.RbacV1().ClusterRoleBindings().Create(context.Background(), crb, metav1.CreateOptions{})
+				if !errors.IsAlreadyExists(err) {
+					util.PanicOnError(err)
```

Comment:
```suggestion
					Expect(err).ToNot(HaveOccurred())
```

Reply:
This portion has not changed from the existing. Doing such a change will require reasoning.

Reply:
won't insist. 
just making a little improvement on the way. 
Could be in a separate PR.

Reply:
here I'm panicking because if this fails, the after test cleanups will not succeed either

Reply:
Not sure I understand the reasoning.
asserts are also panicking under the hood and an explicit panic may not really skip teardown operations. If it does, it will be very bad.